packages/kobotoolbox/test/helper.js

```diff
@@ -0,0 +1,47 @@
+export const responseWithPagination = (url, objects, options) => {
+  const { pageSize, start, limit } = options;
```

Comment:
This shouldn't have pagesize. The helper only ever returns a single page of data, right? So it just needs to take start and limit.

Having a pagesize here just adds confusion and means I can't entirely trust the tests

Reply:
Or am I misunderstanding what this helper does?